ansible/roles/test/files/ptftests/device_connection.py

```diff
@@ -6,7 +6,7 @@
 from pip._vendor.retrying import retry
 logger = logging.getLogger(__name__)
 
-DEFAULT_CMD_EXECUTION_TIMEOUT_SEC = 10
+DEFAULT_CMD_EXECUTION_TIMEOUT_SEC = 300
```

Comment:
Why to increase the default timeout?
For specific cases, where greater timeout is expected/allowed, we can still pass it as an arg:
`execCommand(cmd, timeout=300)`